### PR TITLE
Wax coatings

### PR DESCRIPTION
**Pull request by MicroMinty**
Added wax coatings as a tool mod, and recipe for them.

### CODE REVIEW

File: code/game/objects/items/weapons/tools/mods/_upgrades.dm [hunk 1]
```diff
@@ -267,7 +267,9 @@
 		T.armor.energy += tool_upgrades[UPGRADE_ENERGY_ARMOR]
 	if(tool_upgrades[UPGRADE_BOMB_ARMOR])
 		T.armor.bomb += tool_upgrades[UPGRADE_BOMB_ARMOR]
-
+    if(tool_upgrades[UPGRADE_ITEMFLAGPLUS])
```
Comment (Trilbyspaceclone) [inline code/game/objects/items/weapons/tools/mods/_upgrades.dm:270]: These are spaces not tabs!

Comment (MicroMinty) [reply code/game/objects/items/weapons/tools/mods/_upgrades.dm:270]: im retarded :)

File: code/datums/craft/recipes/tools.dm [hunk 2]
```diff
@@ -257,3 +257,12 @@
 		list(QUALITY_HAMMERING, 15, 150),
 		list(QUALITY_SAWING, 15, 150)
 	)
+//A bucket of wax to be applied to tools
+/datum/craft_recipe/tool/waxcoat
+	name = "tool mod: wax coating"
+	result = /obj/item/tool_upgrade/productivity/waxcoat
+	steps = list(
+			list(CRAFT_MATERIAL, 1, MATERIAL_STEEL)
```
Comment (Trilbyspaceclone) [inline code/datums/craft/recipes/tools.dm:265]: These need , at the end

File: code/datums/craft/recipes/tools.dm [hunk 3]
```diff
@@ -257,3 +257,12 @@
...
 		list(QUALITY_SAWING, 15, 150)
 	)
+//A bucket of wax to be applied to tools
+/datum/craft_recipe/tool/waxcoat
+	name = "tool mod: wax coating"
+	result = /obj/item/tool_upgrade/productivity/waxcoat
+	steps = list(
+			list(CRAFT_MATERIAL, 1, MATERIAL_STEEL),
+			list(QUALITY_WIRE_CUTTING, 10, 50),
+			list(/obj/item/stack/wax, 10, 10),
```
Comment (Trilbyspaceclone) [inline code/datums/craft/recipes/tools.dm:267]:
```suggestion
			list(/obj/item/stack/wax, 10, 10)
```

**Pull request merged**